test/Microsoft.NET.Build.Tests/GivenThereAreDefaultItems.cs

```diff
@@ -24,7 +24,7 @@ public void It_ignores_excluded_folders()
         {
             Action<GetValuesCommand> setup = getValuesCommand =>
             {
-                foreach (string folder in new[] { "bin", "obj", "packages" })
+                foreach (string folder in new[] { "bin", "obj" })
```

Comment:
This seems fine but I'm a tiny bit worried about doing it in combination with the higher-pri change.